clusterloader2/testing/density/config.yaml

```diff
@@ -0,0 +1,116 @@
+automanagedNamespaces: 3
+steps:
+- measurements:
+  - Identifier: TimerGlobal
+    Method: Timer
+    Params:
+      action: start
+  - Identifier: TimerCreate
+    Method: Timer
+    Params:
+      action: start
+  - Identifier: APICallLatency
+    Method: APICallLatency
+    Params:
+      action: reset
+- phases:
+  - namespaceRange:
+      min: 1
+      max: 3
+    replicasPerNamespace: 1
+    tuningSet: TuningSet100
+    objectBundle:
+    - basename: rc1
+      objectTemplatePath: rc.yaml
+      templateFillMap:
+        Replicas: 30
+- measurements:
+  - Identifier: WaitForRunningPods1
+    Method: WaitForRunningPods
+    Params:
+      desiredPodCount: 90
+      labelSelector: phase = phase1
+      timeout: 2m
+- measurements:
+  - Identifier: TimerCreate
+    Method: Timer
+    Params:
+      action: stop
+      label: Pods creation
+  - Identifier: TimerAdditionals
+    Method: Timer
+    Params:
+      action: start
+- phases:
+  - namespaceRange:
+      min: 1
+      max: 3
+    replicasPerNamespace: 1
+    tuningSet: TuningSet5
+    objectBundle:
+    - basename: rc1-add
+      objectTemplatePath: rc.yaml
+      templateFillMap:
+        Replicas: 1
+- measurements:
+  - Identifier: WaitForRunningPods2
+    Method: WaitForRunningPods
+    Params:
+      desiredPodCount: 93
+      labelSelector: phase = phase1
+- measurements:
+  - Identifier: TimerAdditionals
+    Method: Timer
+    Params:
+      action: stop
+      label: Additional pods creation
+  - Identifier: TimerDelete
+    Method: Timer
+    Params:
+      action: start
+- phases:
+  - namespaceRange:
+      min: 1
+      max: 3
+    replicasPerNamespace: 0
+    tuningSet: TuningSet100
+    objectBundle:
+    - basename: rc1
+      objectTemplatePath: rc.yaml
+      templateFillMap:
+        Replicas: 30
+  - namespaceRange:
+      min: 1
+      max: 3
+    replicasPerNamespace: 0
+    tuningSet: TuningSet100
+    objectBundle:
+    - basename: rc1-add
+      objectTemplatePath: rc.yaml
+      templateFillMap:
+        Replicas: 1
+- measurements:
+  - Identifier: APICallLatency
+    Method: APICallLatency
+    Params:
+      action: gather
+      nodeCount: 4
+  - Identifier: TimerDelete
+    Method: Timer
+    Params:
+      action: stop
+      label: Pods deletion
+  - Identifier: TimerGlobal
+    Method: Timer
+    Params:
+      action: stop
+      label: Global execution
+tuningSets:
+- name: TuningSet100
```

Comment:
I'd suggest a better name (like `Uniform100qps`) so that it's obvious to understand when people look at it in  phase definitions. `TuningSet` is obvious and doesn't have to be in the name.